.githooks/pre-push.bash

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
@nolash why is this removed? can you please rebase master?

Reply:
@acud currently this hook fails on my system, and aborts the push.

Reply:
@nolash you could only remove hook from `.git/hooks/pre-push`